server/models/index.js

```diff
@@ -29,6 +29,16 @@ Object.keys(db).forEach(modelName => {
   }
 })
 
+db._connected = async () => {
```

Comment:
We haven't really agreed on conventions, but this function has been given a prefix of `_` which is not used in other projects. I think that's a good convention for functions only used by ther functions in a class, but that's not the case here. One for discussion.

This index file is used to add models to the db, so should probably stay just doing that one thing.
Adding our own function to an object we don't own seems like a bit of a code smell, and risks overriding existing functions.

I'd prefer the `isConnected` function to be in its own file, using the 'db' exposed by this file to access sequelize.
It's easier to mock an `isConnected` function that returns true or false too.

Reply:
I agree. Moved the code out, stolen John's code from the Payment service as it did exactly what you suggested.